.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+## Why was this change made?
+
+
+
+## Was the usage documentation (e.g. wiki, README, queue or DB specific README) updated?
```

Comment:
```suggestion
## Was the usage documentation (e.g. wiki, README) updated?
```

Reply:
i think this was knocked off from pres cat's template, but queue and DB specific READMEs are specific to prescat, so should prob drop those.

if you accept this suggestion, you can keep from having to re-push (and we can squash my suggestion into your commit).